pytorch/inference/docker/1.9/py3/cu111/Dockerfile.gpu

```diff
@@ -0,0 +1,204 @@
+FROM nvidia/cuda:11.1.1-base-ubuntu20.04
+
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+ARG PYTHON=python3
+ARG PYTHON_VERSION=3.8.10
+
+# PyTorch Binaries
+ARG PT_INFERENCE_URL=https://pytorch-ei-binaries.s3.us-west-2.amazonaws.com/r1.9.0_inference/20210617-030314/46d3d4fb3c4208f6fcdf1e6aca78ed1fc9155236/gpu/torch-1.9.0-cp38-cp38-manylinux1_x86_64.whl
+ARG PT_TORCHVISION_URL=https://pytorch-ei-binaries.s3.us-west-2.amazonaws.com/torchvision_build/0.10.0/gpu/torchvision-0.10.0-cp38-cp38-linux_x86_64.whl
+ARG PT_TORCHAUDIO_URL=https://pytorch-ei-binaries.s3.us-west-2.amazonaws.com/torchaudio_build/0.9.0/gpu/torchaudio-0.9.0-cp38-cp38-linux_x86_64.whl
+
+ARG OPEN_MPI_VERSION=4.0.1
+ARG EFA_PATH=/opt/amazon/efa
+ARG TS_VERSION=0.4.0
+ARG CUBLAS_VERSION=11.3.0.106
+
+ARG DEBIAN_FRONTEND=noninteractive
+# See http://bugs.python.org/issue19846
+ENV LANG C.UTF-8
+ENV LD_LIBRARY_PATH /opt/conda/lib/:$LD_LIBRARY_PATH
+ENV PATH /opt/conda/bin:$PATH
+ENV SAGEMAKER_SERVING_MODULE sagemaker_pytorch_serving_container.serving:main
+ENV TEMP=/home/model-server/tmp
+ENV TZ=America/New_York
```

Comment:
Referring to the concern raised on PT Training PR https://github.com/aws/deep-learning-containers/pull/1132#discussion_r645806680, 
Why the TZ config is necessary as DLC is meant to be used in any region/timezone? would the `ENV DEBIAN_FRONTEND=noninteractive` config sufficient?

Reply:
Thanks for the feedback, removed TZ.